docs/node/hello-world.md

```diff
@@ -0,0 +1,81 @@
+# Hello world
```

Comment:
Let's create a "Tutorials" category and make this comparable to the other "Hello World from scratch" tutorials in the other SDKs.